test/Microsoft.NET.Build.Tests/AppHostTests.cs

```diff
@@ -167,7 +136,7 @@ public void It_codesigns_an_app_targeting_osx(string targetFramework, string rid
                 .WithTargetFramework(targetFramework);
 
             var buildCommand = new BuildCommand(testAsset);
-            var buildArgs = new List<string>() { $"/p:RuntimeIdentifier={rid}" };
+            var buildArgs = new List<string>() { $"/p:RuntimeIdentifier={rid}", $"/p:_EnableMacOSCodeSign={shouldSign}" };
```

Comment:
We should keep test cases for the default (that is, not explicitly setting `_EnableMacOSCodeSign`) relying on the targets to set the right thing, since that should be the normal user scenario.

Reply:
Same for the publish tests.